client/components/tiles.jsx

```diff
@@ -97,6 +98,15 @@ class Tile extends Component {
     });
   }
 
+  role() {
```

Comment:
In order to self-document code, consider using a more descriptive name such as renderDeleteForTeacher or similar.